client/state/selectors/is-requesting-jetpack-onboarding-settings.js

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import { getRequest } from 'state/selectors';
+import { requestJetpackOnboardingSettings } from 'state/jetpack-onboarding/actions';
+
+/**
+ * Returns true if we are currently making a request to fetch the Jetpack settings. False otherwise
+ * Returns null if the status for the queried site is unknown.
```

Comment:
It doesn't seem that this one will ever return a `null` I think, so we should be safe to remove this line.